### ISSUE TITLE
enroll page: Add more metadata

### ISSUE THREAD

**Issue by trombonekenny** [PrairieLearn/PrairieLearn#3149]
Add another column to the enroll page, perhaps just for admins or course owners, to show "why" the option is visible the way it is. Something like:

![image](https://user-images.githubusercontent.com/10765199/93615075-8e8a6080-f998-11ea-90ad-486390d81b41.png)

Maybe that extra info is OK for everyone to see, but it would at least help me to figure out/support why admin (or maybe just mussulma@illinois.edu) sees so many courses. I suspect it's because I'm an instructor in a copied example course.

**Comment by tbretl** [comment]
Agreed. This issue should be revisited though after the new permissions scheme is reviewed and merged because many of these problems will go away and because root causes will change (e.g., "an instructor in a copied example course" will no longer be a thing).

**Comment by trombonekenny** [comment]
The PR that pushed the new course staff permissions added a `users_is_instructor_in_course(u.user_id, c.id) IS FALSE` to the enroll page query, essentially stopping admins from seeing any content on that page. We should remove that check so admins can see what a normal enrollee might see on the page - with potentially even more data displayed to admins as described in the original issue.

**Comment by reteps** [comment]
Closing as there is no longer an enroll page.
